patches/0007-add-support-for-logging-used-Windows-APIs.patch

```diff
@@ -4,42 +4,29 @@ Date: Tue, 19 Mar 2024 16:48:18 +0100
 Subject: [PATCH] add support for logging used Windows APIs
 
 ---
- src/runtime/os_windows.go                    |   1 +
- src/runtime/syscall_windows.go               |   1 +
- src/runtime/syscalltrace_generate_windows.go |  50 +++++++
- src/runtime/syscalltrace_windows.go          | 131 +++++++++++++++++++
- src/runtime/syscalltrace_windows_test.go     |  81 ++++++++++++
- src/runtime/zsyscalltrace_windows.go         |  60 +++++++++
- 6 files changed, 324 insertions(+)
+ src/runtime/os_windows.go                     |   1 +
+ src/runtime/syscalltrace_generate_windows.go  |  50 +++++++
+ src/runtime/syscalltrace_windows.go           | 131 ++++++++++++++++++
+ src/runtime/syscalltrace_windows_test.go      |  81 +++++++++++
+ src/runtime/zsyscalltrace_windows.go          |  60 ++++++++
+ 5 files changed, 323 insertions(+)
  create mode 100644 src/runtime/syscalltrace_generate_windows.go
  create mode 100644 src/runtime/syscalltrace_windows.go
  create mode 100644 src/runtime/syscalltrace_windows_test.go
  create mode 100644 src/runtime/zsyscalltrace_windows.go
 
 diff --git a/src/runtime/os_windows.go b/src/runtime/os_windows.go
-index 88d730aa026a7f..53e3332d9fa77f 100644
+index ab4e165baecfb0..608ee8bc962a0c 100644
 --- a/src/runtime/os_windows.go
 +++ b/src/runtime/os_windows.go
-@@ -222,6 +222,7 @@ func windowsFindfunc(lib uintptr, name []byte) stdFunction {
+@@ -220,6 +220,7 @@ func windowsFindfunc(lib uintptr, name []byte) stdFunction {
  	if name[len(name)-1] != 0 {
  		throw("usage")
  	}
 +	syscallTrace(lib, &name[0])
- 	f := stdcall2(_GetProcAddress, lib, uintptr(unsafe.Pointer(&name[0])))
+ 	f := stdcall(_GetProcAddress, lib, uintptr(unsafe.Pointer(&name[0])))
  	return stdFunction(unsafe.Pointer(f))
  }
-diff --git a/src/runtime/syscall_windows.go b/src/runtime/syscall_windows.go
```

Comment:
This function go moved to `src/syscall/dll_windows.go` via https://github.com/golang/go/commit/ef4054978692bd934eb575d0027c1c5476af9931

I don't think we can use `syscallTrace` in that file so I've skipped adding it